AlphaWalletTests/Tokens/Coordinators/GetENSOwnerCoordinatorTests.swift

```diff
@@ -38,18 +41,21 @@ class GetENSOwnerCoordinatorTests: XCTestCase {
         expectations.append(expectation)
         let ensName = "ethereum.eth"
         let server = makeServerForMainnet()
+        var fetchedAddress: AlphaWallet.Address?
+        var fetchedError: Error?
         firstly {
             ENSResolver(server: server).getENSAddressFromResolver(forName: ensName)
         }.done { address in
-            if address.sameContract(as: "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe") {
```

Comment:
Would it work and be simpler if we move the `expectation.fulfill()` to a `.ensure {}` block at the end of the promise chain so it's always executed.

Reply:
Put in `expectation.fufilled()` in an ensure block.